### PR TITLE
Fix python-mip-sklearn build

### PR DESCRIPTION
**Pull request by Marigold**

### CODE REVIEW

File: python-mip-sklearn/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,42 @@
-FROM hbpmip/python-base-build:0.4.4
+FROM hbpmip/python-mip:0.4.0
```
Comment (ludovicc) [inline python-mip-sklearn/Dockerfile]: Ideally we should keep python-base-build as base image

File: python-mip-sklearn/Dockerfile [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
+
 FROM hbpmip/python-mip:0.4.0
 
 COPY requirements.txt /
 COPY docker-entrypoint.sh /
 
 RUN pip install -r /requirements.txt
 
+RUN apt-get update
+RUN apt-get install -y git
```
Comment (ludovicc) [inline python-mip-sklearn/Dockerfile]: Two operations and no cleanup, this increases needlessly the size of the Docker image

**Comment by ludovicc** [comment]
Codacy review:

python-mip-sklearn/Dockerfile
Delete the apt-get lists after installing something


RUN apt-get update

Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`


RUN apt-get install -y git

Avoid additional packages by specifying `--no-install-recommends`


RUN apt-get install -y git

Use WORKDIR to switch to a directory

RUN cd sklearn_to_pfa && python setup.py install

**Comment by Marigold** [comment]
Thanks, I've seen the codacy review, but didn't have time to fix it yet. I'll create a new PR fixing issues you mentioned.

**Pull request merged**